test/tests/functional/pbs_node_jobs_restart_multinode.py

```diff
@@ -57,8 +57,10 @@ def test_restart_hosts_resume(self):
         momB = self.moms.values()[1]
 
         # Make sure moms are running with -p flag
-        momA.restart(args=['-p'])
-        momB.restart(args=['-p'])
+        momA.stop(sig='-INT')
```

Comment:
@sakshamgarg  does the mom.stop() validate, if mom has indeed stopped ? if not please consider adding a validation check after each start and stop.

Reply:
Yes @Shrini-h. The stop function validates whether MoM service has stopped or not.